pkg/list/list_stacks_test.go

```
@@ -0,0 +1,54 @@
+	configAndStacksInfo := schema.ConfigAndStacksInfo{}
+
+	cliConfig, err := cfg.InitCliConfig(configAndStacksInfo, true)
+	assert.Nil(t, err)
+
+	stacksMap, err := e.ExecuteDescribeStacks(cliConfig, "", nil, nil,
+		nil, false, false, false)
+	assert.Nil(t, err)
+
+	output, err := FilterAndListStacks(stacksMap, "")
+	assert.Nil(t, err)
+	dependentsYaml, err := u.ConvertToYAML(output)
```

golangci-lint (1.61.0)

ineffectual assignment to err

(ineffassign)